src/cockpit/bridge.py

```diff
@@ -62,12 +62,12 @@ def export(self, path: str, obj: bus.BaseObject) -> None:
 class Bridge(Router, PackagesListener):
     internal_bus: InternalBus
     packages: Optional[Packages]
-    bridge_rules: List[JsonObject]
```

Comment:
I think this was a bug in my mega-JSON-patch: this ought to be a `List[BridgeConfig]` now, no?

Reply:
Indeed it should, fixed. But as a BridgeConfig is a JsonObject, mypy was still happy.

Reply:
Yup, sorry, confusion with mypy not actually running locally. All fixed now.